h/static/scripts/test/root-thread-test.js

```diff
@@ -199,30 +200,52 @@ describe('rootThread', function () {
       assert.isDefined(threadFilterFn({annotation: annotation}));
     });
 
-    it('generates a thread filter function to match notes', function () {
+    it('filter matches only notes when Notes tab is selected', function () {
       fakeBuildThread.reset();
-      fakeBuildThread.annotation = {target: [{}]};
 
       fakeAnnotationUI.state = Object.assign({}, fakeAnnotationUI.state,
-        {selectedTab: 'note'});
+        {selectedTab: uiConstants.TAB_NOTES});
 
       rootThread.thread(fakeAnnotationUI.state);
       var threadFilterFn = fakeBuildThread.args[0][1].threadFilterFn;
 
-      assert.isTrue(threadFilterFn(fakeBuildThread));
+      assert.isTrue(threadFilterFn({annotation: {target: [{}]}}));
     });
 
-    it('generates a thread filter function for annotations, when all annotations are of type notes', function () {
+    it('filter matches only orphans when Orphans tab is selected', function () {
       fakeBuildThread.reset();
-      fakeBuildThread.annotation = {target: [{}]};
 
       fakeAnnotationUI.state = Object.assign({}, fakeAnnotationUI.state,
-        {selectedTab: 'annotation'});
+        {selectedTab: uiConstants.TAB_ORPHANS});
 
       rootThread.thread(fakeAnnotationUI.state);
       var threadFilterFn = fakeBuildThread.args[0][1].threadFilterFn;
 
-      assert.isFalse(threadFilterFn(fakeBuildThread));
+      assert.isTrue(threadFilterFn({annotation: {$orphan: true}}));
+    });
+
+    it('filter matches only notes when Annotations tab is selected', function () {
+      fakeBuildThread.reset();
+
+      fakeAnnotationUI.state = Object.assign({}, fakeAnnotationUI.state,
+        {selectedTab: uiConstants.TAB_ANNOTATIONS});
+
+      rootThread.thread(fakeAnnotationUI.state);
+      var threadFilterFn = fakeBuildThread.args[0][1].threadFilterFn;
+
+      assert.isFalse(threadFilterFn({annotation: {target: [{}]}}));
+    });
+
+    it('filter matches only orphans when Annotations tab is selected', function () {
```

Comment:
This test description has the same problem as the above test that the description doesn't match the assertion. The test checks that the filter _does not match_ orphans when the Annotations tab is selected.